charts/brig/templates/configmap.yaml

```diff
@@ -26,6 +26,9 @@ data:
         host: {{ .cassandra.host }}
         port: 9042
       keyspace: brig
+      {{- if hasKey .cassandra "filterNodesByDatacentre" }}
```

Comment:
This syntax is wrong

```
  Error: template: wire-server/charts/spar/templates/deployment.yaml:27:31: executing "wire-server/charts/spar/templates/deployment.yaml" at <include (print .Template.BasePath "/configmap.yaml") .>: error calling include: template: wire-server/charts/spar/templates/configmap.yaml:23:20: executing "wire-server/charts/spar/templates/configmap.yaml" at <.cassandra>: wrong type for value; expected map[string]interface {}; got interface {}
  ```
  
  try with `helm template` to get the syntax right?

Reply:
I pushed a commit to fix the issue.

Reply:
Too much copy-pasta for me!